scripts/referrer-user-count.ts

```diff
@@ -0,0 +1,92 @@
+import { stringify } from 'csv-stringify/sync'
```

Comment:
Can this file be camelCased? That's the convention we've been going with for files/script names.